adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -1888,4 +1889,29 @@ abstract class AvroGenomicRDD[T <% IndexedRecord: Manifest, U <: Product, V <: A
   def saveAsParquet(filePath: java.lang.String) {
     saveAsParquet(new JavaSaveArgs(filePath))
   }
+
+  def saveAsPartitionedParquet(filePath: String,
+                               blockSize: Int = 128 * 1024 * 1024,
+                               pageSize: Int = 1 * 1024 * 1024,
+                               compressCodec: CompressionCodecName = CompressionCodecName.GZIP,
+                               disableDictionaryEncoding: Boolean = false,
+                               partitionSize: Int = 1000000) {
+    log.warn("Saving directly as Hive-partitioned Parquet from SQL. " +
+      "Options other than compression codec are ignored.")
+    import org.apache.spark.sql.functions._
+    val genomicBinSize = partitionSize
```

Comment:
I wouldn't dupe the variable; I would prefer `s/genomicBinSize/partitionSize/g` throughout this function.